index.bs

```diff
@@ -10568,6 +10552,39 @@ class MyProcessor extends AudioWorkletProcessor {
 }
 </xmp>
 
+<h6 id="audioworkletprocess-callback-parameters" callback lt="AudioWorkletProcessCallback()">
+Callback {{AudioWorkletProcessCallback}} Parameters
+</h6>
+The following describes the parameters to the {{AudioWorkletProcessCallback}} function.
+
+In general, the {{AudioWorkletProcessCallback/inputs!!argument}} and
+{{AudioWorkletProcessCallback/outputs!!argument}} arrays will be reused
+between calls so that no memory allocation is done.  However, if the
+topology changes, because, say, the number of channels in the input or the
+output changes, new arrays are reallocated.  New arrays are also
+reallocated if any part of the
+{{AudioWorkletProcessCallback/inputs!!argument}} or
+{{AudioWorkletProcessCallback/outputs!!argument}} arrays are
+transferred.
+
+<dl dfn-type=argument dfn-for="AudioWorkletProcessCallback">
+	: {{AudioWorkletProcessCallback/inputs!!argument}}, of type <code>{{FrozenArray}}&lt;{{FrozenArray}}&lt;{{Float32Array}}&gt;&gt;</code>
+	:: The input audio buffer from the incoming connections provided by the user agent. <code class="nobreak">inputs[n][m]</code> is a {{Float32Array}} of audio samples for the \(m\)th channel of the \(n\)th input. While the number of inputs is fixed at construction, the number of channels can be changed dynamically based on [=computedNumberOfChannels=].
+
+		If there are no [=actively processing=] {{AudioNode}}s connected to the \(n\)th input of the {{AudioWorkletNode}} for the current render quantum, then the content of <code>inputs[n]</code> is an empty array, indicating that zero channels of input are available. This is the only circumstance under which the number of elements of <code>inputs[n]</code> can be zero.
+
+	: {{AudioWorkletProcessCallback/outputs!!argument}}, of type <code>{{FrozenArray}}&lt;{{FrozenArray}}&lt;{{Float32Array}}>></code>
+	:: The output audio buffer that is to be consumed by the user agent. <code class="nobreak">outputs[n][m]</code> is a {{Float32Array}} object containing the audio samples for \(m\)th channel of \(n\)th output. Each of the {{Float32Array}}s are zero-filled. The number of channels in the output will match [=computedNumberOfChannels=] only when the node has a single output.
+
+	: {{AudioWorkletProcessCallback/parameters!!argument}}, of type {{object}}
+	:: An [=ordered map=] of <var>name</var> → <var>parameterValues</var>. <code>parameters["<var>name</var>"]</code> returns <var>parameterValues</var>, which is a {{FrozenArray}}&lt;{{Float32Array}}> with the automation values of the <var>name</var> {{AudioParam}}.
+
+		For each array, the array contains the [=computedValue=] of the parameter for all frames in the [=render quantum=]. However, if no automation is scheduled during this render quantum, the array MAY have length 1 with the array element being the constant value of the {{AudioParam}} for the [=render quantum=].
+
+		*Insert text here about how these aren't freshly allocated each time, unless the object is transferred.*
```

Comment:
This still says "Insert text here...".

Reply:
A reminder that this still needs to be fixed.  Either here or in the part about inputs/outputs.